language/src/bin.rs

```diff
@@ -51,43 +51,6 @@ fn main() {
 
     let environment: HashMap<String, String> = env::vars().collect();
 
```

Comment:
Do you want to keep this code around somewhere or file a follow-up issue to get the functionality back?

Reply:
I think the code needs a lot of rewriting, since we need to retrieve what files to update, and do multiple comparisions. I am not sure whether this should be done by the backend or some helper tool / manually, it depends on how much it is being used.

Reply:
Yes, a lot of the code here needs to be cleaned up ... 
Having an issue to discuss and follow-up is good in any case.